error-handling/src/main/java/com/palantir/remoting2/errors/SerializableErrorToExceptionConverter.java

```diff
@@ -46,6 +47,7 @@ private SerializableErrorToExceptionConverter() {}
 
     private static final ObjectMapper MAPPER = new ObjectMapper()
             .registerModule(new GuavaModule())
+            .registerModule(new Jdk8Module())
```

Comment:
probably worth making this ObjectMapper and the one we use for request/response the same, at least in configuration, so that there's no confusion.